tst/teststandard/bugfix.tst

```diff
@@ -1082,8 +1082,8 @@ gap> StructureDescription(testG(8,2));
 "((C8 x C2) : C2) : C2"
 gap> StructureDescription(testG(8,3));
 "C3 x QD16"
-gap> StructureDescription(testG(8,4));
-"((C16 x C2) : C2) : C2"
+gap> Length(StructureDescription(testG(8,4)));
```

Comment:
Still not going to work, because without packages it returns "(C16 x C4) : C2", which has different length. (Could check for length>0, but I like the more stable nice option.) Also testG(8,2) is problematic. The nice option, though, would give a stable result, and further, it would more likely reveal problems with existing code, as in #973 I found that ````Centralizer```` cannot necessarily be computed in testG(8,2) (but this may be intended, I am not sure).